pkg/operator/starter.go

```diff
@@ -53,7 +53,7 @@ import (
 	"github.com/openshift/library-go/pkg/operator/resourcesynccontroller"
 	"github.com/openshift/library-go/pkg/operator/revisioncontroller"
 	"github.com/openshift/library-go/pkg/operator/staleconditions"
-	"github.com/openshift/library-go/pkg/operator/staticpod/controller/guard"
+	staticpodcommon "github.com/openshift/library-go/pkg/operator/staticpod/controller/common"
```

Comment:
move non-generated changes into a separate commit

Reply:
The version bump broke the system and I needed to fix it. So I should create a commit that contains the update (even though it isn't working in itself) and a fix for the update in a separate commit?

Reply:
yes